### ISSUE TITLE
Add ability to read Sollet-exported private key instead of requiring base58 encoding

### ISSUE THREAD

**Issue by skynetcapital** [skynetcapital/solanaj#6]
Add easy ability to read Sollet-exported private key into your SolanaJ wallet. SolanaJ is currently configured to only read base58 encoded private key. See the following pseudocode for an example.
```
import base58
privkey = [123,...]  # from Sollet
print(base58.b58encode(bytearray(privkey)))
```

**Comment by skynetcapital** [comment]
Will be creating a static method called `Account.fromJson(String)`.

**Comment by skynetcapital** [comment]
Fixed with 2a39174120f84700d2e072017dda8d324719d7fc. Closing.
